src/types.ts

```diff
@@ -26,6 +25,30 @@ export interface RawDep {
 
 export type DiffType = 'major' | 'minor' | 'patch' | 'error' | null
 
+// @types/pacote uses "import = require()" syntax which is not supported by unbuild
+// So instead of using @types/pacote, we declare the type definition with only fields we need
+export interface Packument {
```

Comment:
Previously, `pacote` was a direct dependency.

If `pacote` is removed, `@types/pacote` can not live alone, and `import = require()` would cause the build to fail, hence the manual declaration.